views/view.html

```diff
@@ -16,6 +16,10 @@
 <meta name="google-site-verification" content="{{ data.global.seoGoogleVerificationId }}" />
 {% endif %}
 
+{% if data.page._seoCanonical  %}
+	<link rel="canonical" href="{{data.baseUrl}}{{data.page._seoCanonical.slug }}" />
```

Comment:
I think `data.page._seoCanonical._url` should be enough here. We count on `baseUrl` being set on the project in cases like this.

Reply:
@woodbrearlham This still needs to be updated to use `_url`.